modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro

```diff
@@ -0,0 +1,59 @@
+definition {
+
+	macro configurationOpenGraph {
+		AssertElementPresent(
+			key_title = "Open Graph",
+			locator1 = "Header#H4_TITLE");
+
+		if (isSet(customTitle)) {
+			var checkboxName = "Use Custom Title";
+
+			if (IsNotChecked(locator1 = "Checkbox#ANY_CHECKBOX")) {
+				Click(locator1 = "Checkbox#ANY_CHECKBOX");
+
+				Type(
+					locator1 = "TextInput#OPEN_GRAPH_CUSTOM_TITLE",
+					value1 = "${customTitle}");
+			}
+			else {
```

Comment:
Remove this statement. A user should not be able to directly type into the text box unless the checkbox is checked. That's a bug